lib/paypalhttp/serializers/form_encoded.rb

```diff
@@ -1,11 +1,12 @@
-require 'cgi'
+require 'uri'
 
 module PayPalHttp
   class FormEncoded
     def encode(request)
       encoded_params = []
+      parser = URI::Parser.new()
```

Comment:
I'm not a rubyist, so I don't have a lot of context on this, but is creating a new parser for each time `encode` is called the correct thing to do? Or should the module have a single parser that is re-used?

Again, I'm not sure what the best option is for ruby. This is a genuine question.

Reply:
That is a good point. I am unsure of how Ruby handles the creation of this parser. To ensure best practice, we are moving the parser to be initialized within the class and then referenced